external/CMakeLists.txt

```diff
@@ -139,18 +139,38 @@ target_link_libraries(yaml INTERFACE yaml-cpp)
 add_library(rpcs3 INTERFACE)
 target_include_directories(rpcs3 INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/rpcs3/include")
 
-add_library(discord-rpc INTERFACE)
-if (USE_DISCORD_RICH_PRESENCE)
-	target_compile_definitions(discord-rpc INTERFACE DISCORD_RPC=NDEBUG)
-endif()
-target_include_directories(discord-rpc INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/discord-rpc/include")
+message(STATUS "Downloading discord gamesdk...")
+file(DOWNLOAD https://dl-game-sdk.discordapp.net/2.5.6/discord_game_sdk.zip
```

Comment:
We probably can build `boost` with `CI`, publish it on the release page and add the download URL here.